CONTRIBUTING.md

```diff
@@ -61,7 +61,8 @@ To run all the tests:
 
 Treat commit messages as an email message that describes what you changed and why.
 
-The first line of the commit log must be treated as as an email
+The first line of the commit log 
```

Comment:
I think the current text was better than your modification, here.

It could be simplified to:

The first line of the commit log is the email subject.

That removes `line` being repeated, and strictly speaking the email subject line must have the prefix `Subject:` , which is not needed in the commit log first line.